models/users.js

```diff
@@ -82,6 +82,28 @@ const fetchUsers = async (query) => {
   }
 };
 
+const queryUser = async (query) => {
+  try {
+    let call = userModel;
+    Object.keys(query).forEach((key) => {
+      // eslint-disable-next-line security/detect-object-injection
+      call = call.where(key, "==", query[key]);
```

Comment:
It's not good practice to use dynamic column name selection. checkout and read xss injection